corehq/apps/app_manager/models.py

```diff
@@ -1048,6 +1048,9 @@ class FormBase(DocumentSchema):
     # computed datums IDs that are allowed in endpoints
     function_datum_endpoints = StringListProperty()
 
+    submit_label = DictProperty(default={})
+    submit_notification_label = DictProperty(default={})
```

Comment:
Can you use `LabelProperty` for this just for clarity?  (It's basically the same thing)
https://github.com/dimagi/commcare-hq/blob/6e7975aec6094e6c9593415ddf3c7b67e977de31/corehq/apps/app_manager/models.py#L206-L207